core/alts/alts_manager.py

```diff
@@ -19,14 +19,14 @@ def inject(self, registry):
     def start(self):
         pass
 
-    def get_alts(self, char_id):
+    def get_alts(self, char_id, status = self.VALIDATED):
         sql = "SELECT p.*, a.group_id, a.status FROM player p " \
               "LEFT JOIN alts a ON p.char_id = a.char_id " \
               "WHERE p.char_id = ? OR a.group_id = (" \
               "SELECT group_id FROM alts WHERE status >= ? AND char_id = ?) " \
-              "ORDER BY a.status DESC, p.level DESC"
+              "ORDER BY status = ? DESC, a.status DESC, p.level DESC"
```

Comment:
this is not necessary since MAIN = 2, VALIDATED = 1, and UNVALIDATED = 0, it will already sort main as first